mindmeld/tokenizer.py

```diff
@@ -50,60 +53,61 @@ def _init_regex(self):
 
         exception_chars = "\@\[\]\|\{\}'"  # noqa: W605
 
-        # This is more of a tactical fix at this moment. Will probably need a more generic way
-        # to handle all currency symbols.
-        currency_symbols = "$¥"
+        # fetches all currency symbols in unicode
+        currency_symbols = u"".join(
+            chr(i) for i in range(0xFFFF) if unicodedata.category(chr(i)) == "Sc"
+        )
```

Comment:
Could you write a comment explaining what the logic is here?

Reply:
Added, let me know if this works

Reply:
Since this logic is happening twice, I think it makes sense just to add this is constants.py